### ISSUE TITLE
Warning for 'more than 80 characters on line' is not catching all instances

### ISSUE THREAD

**Issue by pccowboy** [rodjek/puppet-lint#159]
Similar to my previous issue, I have a file with the following content:

```
# Class: jenkins::add_config
#
# This define is a convenience function to put a jenkinizer config into
#  the correct place to work on a jenkins instance
#
# Parameters:
# * none
#
# Actions:
# * copies the passed config to /home/jenkins/jenkinizer/config
#
# Requires:
# * name of the configuration
#
# Sample Usage:
# Create the config for each server in it's own
# directory tree under the 'files/jenkins' mount for the
# vertical.  Keep all configuration artifacts needed in each
# tree, do not assume anything will happen magically for you.
#
define jenkins::add_config (
  $configuration_name
) {

  file { "jenkinizer-config-${spi_unified_provisioner::params::molecule}-${configuration_name}":
    ensure  => directory,
    mode    => '0644',
    owner   => jenkins,
    group   => jenkins,
    path    => "${jenkins::home}/jenkinizer/config",
    recurse => true,
    source  => "puppet:///modules/${spi_unified_provisioner::params::molecule}/jenkins/${configuration_name}",
  }

}
```

and I get 

```
WARNING: line has more than 80 characters on line 25
```

There should be a warning on line 32 as well

If I put some whitespace on line 34 (just to see if maybe something is cutoff in my window), I see:

```
WARNING: line has more than 80 characters on line 25
ERROR: trailing whitespace found on line 34
```

If it makes any difference, my command is being executed via an Aptana Ruble, that looks like this:

```
    cmd.invoke do |context|
      result = `/Users/dswift/.rvm/gems/ruby-1.8.7-p358/bin/puppet parser validate --color false --render-as s --modulepath=/Users/dswift/source/tools/ops/puppet/modules #{ENV['TM_FILEPATH']}`
      result += "\n"
      result += `/Users/dswift/.rvm/gems/ruby-1.8.7-p358/bin/puppet-lint #{ENV['TM_FILEPATH']}`
      result
    end
```
